addons/website_links/static/src/interactions/website_links_charts.js

```diff
@@ -0,0 +1,233 @@
+                selector: "#last_month_countries_charts",
+                data: this.lastMonthClicksByCountry,
+            },
+            {
+                selector: "#last_week_countries_charts",
+                data: this.lastWeekClicksByCountry,
+            },
+        ];
+
+        const width = this.el.querySelector(pieChartsData[0].selector).scrollWidth;
+        for (const pieChartData of pieChartsData) {
+            const chart = this.createPieChart(pieChartData.selector, pieChartData.data);
+
+            // Adapt chart height for when many country labels
+            chart.canvas.parentNode.style.height = `${Math.max(
+                pieChartData.data.length * (width > 750 ? 1 : 2),
+                20
+            )}em`;
+        }
+    }
+
+    createPieChart(selector, data) {
+        const labels = [];
+        const formattedData = [];
+        for (const countryData of data) {
+            const countryName = countryData["country_id"]
+                ? countryData["country_id"][1]
+                : _t("Undefined");
+            labels.push(`${countryName} (${countryData["__count"]})`);
+            formattedData.push(countryData["__count"]);
+        }
+
+        const containerEl = this.el.querySelector(selector);
+        containerEl.querySelector(".title").textContent = `${formattedData.length} ${_t(
+            "countries"
+        )}`;
+        const config = {
+            type: "pie",
+            data: {
+                labels,
+                datasets: [
+                    {
+                        data: formattedData,
+                        label: _t("# of clicks"),
+                    },
+                ],
+            },
+            options: {
+                aspectRatio: 2,
+                responsive: true,
+                maintainAspectRatio: false,
+            },
+        };
+
+        const canvasEl = containerEl.querySelector("canvas");
+        const context = canvasEl.getContext("2d");
+        return new Chart(context, config);
+    }
+
+    createLineChart(selector, beginDate, endDate, dates) {
+        const containerEl = this.el.querySelector(selector);
+        beginDate = beginDate.startOf("day");
+        endDate = endDate.startOf("day");
+        if (beginDate.toISO() === endDate.toISO()) {
+            endDate = endDate.plus({ days: 1 });
+        }
+        const numDays = endDate.diff(beginDate).as("days");
+        let totalClicks = 0;
+        const data = [];
+        const labels = [];
+        let currentDate = beginDate;
+        for (let i = 0; i <= numDays; i++) {
+            const dateKey = currentDate.toFormat("yyyy-MM-dd");
+            labels.push(dateKey);
+            const numClicks = dateKey in dates ? dates[dateKey] : 0;
+            data.push(numClicks);
+            totalClicks += numClicks;
+            currentDate = currentDate.plus({ days: 1 });
+        }
+
+        containerEl.querySelector(".title").textContent = `${totalClicks} ${_t("clicks")}`;
```

Comment:
Try doing this in a `t-out` in `dynamicContent`... maybe you'll need another parameter to give a variable name to store the value, or return `totalClicks` to the caller that will keep track of it in a dedicated variable.